playbook/app/pb_kits/playbook/pb_typeahead/_typeahead.jsx

```diff
@@ -57,6 +67,7 @@ const Typeahead = ({ loadOptions = noop, getOptionLabel, id, getOptionValue, cre
       Option,
       Placeholder,
       ValueContainer,
+      ...components
```

Comment:
This is the actual magic that allows us to overload our default components for the one created in the example.